### PR TITLE
Add .toLowerCase to values to fix searching

### PR DESCRIPTION
**Pull request by natesheridan**
## [PR for What's Cookin 🍳]

*(assume (default) means no changes or NA)*

**What (if any) features are you implementing?**

 - (default)
 
**What (if anything) did you refactor?**

 - Made sure that the Filter methods inside of the recipeRepo class don't fail because of case sensitivity

**Were there any issues that arose?** 

 - Nothing was working regarding our search bar - this fixes it.

**Is there anything needs to be prioritized?**

 - (default)

**Is there anything that you need from your teammates?**

 - (default)

**GOALS?**

 - (default)

### CODE REVIEW

File: src/classes/RecipeRepository.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ class RecipeRepository {
...
     const filteredRecipes = this.recipeData.filter((recipe) => {
-      return recipe.tags.includes(tagName)
+      return recipe.tags.toLowerCase().indexOf(tagName.toLowerCase())
     })
     return filteredRecipes;
   }
   filterByName(recipeName){
     const filteredRecipes = this.recipeData.filter((recipe) => {
-      return recipe.name.includes(recipeName)
+      return recipe.name.toLowerCase().includes(recipeName.toLowerCase())
```
Comment (reneeswick) [inline src/classes/RecipeRepository.js:13]: YES! Great catch that I did not test for in test suites.

**Review by reneeswick — APPROVED** [review]
This edge case really snuck up on us! Great fix here. Thanks for your hard work!
